platform/spire/debian-iso/deps.bzl

```diff
@@ -1,8 +1,8 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
 
-VERSION = '20190702+deb10u3'
-RELEASE = 'buster'
-MINI_ISO_HASH = '26b6f6f6bcb24c4e59b965d4a2a6c44af5d79381b9230d69a7d4db415ddcb4cd'
+VERSION = "20190702+deb10u3"
+RELEASE = "buster"
+MINI_ISO_HASH = "26b6f6f6bcb24c4e59b965d4a2a6c44af5d79381b9230d69a7d4db415ddcb4cd"
```

Comment:
ah, right... I forgot last time that this goes is a bazel definition file, not a shell-script.  so it does want double-quotes :stuck_out_tongue: